common/src/main/java/org/broadleafcommerce/common/cache/Log4jStatisticsServiceLogAdapter.java

```diff
@@ -17,8 +17,10 @@
  */
 package org.broadleafcommerce.common.cache;
 
-import org.apache.log4j.Level;
-import org.apache.log4j.LogManager;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.config.Configurator;
 
 /**
```

Comment:
There is already a class Log4j2StatisticsServiceLogAdapter. Can we just delete this one?